folding/src/lib.rs

```diff
@@ -262,16 +266,16 @@ impl<'a, CF: FoldingConfig> FoldingScheme<'a, CF> {
 /// of other element. This type represents that, allowing to also recognize
 /// which case is present.
 #[derive(Debug, Clone)]
-pub enum Alphas<F: Field> {
-    Powers(F, Rc<AtomicUsize>),
-    Combinations(Vec<F>),
+pub enum Alphas<G: AffineCurve> {
```

Comment:
Why not keeping the field?

Reply:
Oh, that's just a version I had so that the types of the generic checker would match. But now I am trying to switch it for field as it was in your PR and it also seems to compile just fine. So I will set it to field, as it makes more sense that way.